src/commcare_cloud/fab/diff_templates/console.txt.j2

```diff
@@ -29,6 +29,16 @@ Changelogs:
 {% endfor %}
 {%- endif %}
 
+{%- if not (is_dimagi_env and maintenance_prs.error) and maintenance_prs.maintenance_prs -%}
```

Comment:
shouldn't this be `or`:
```suggestion
{%- if not (is_dimagi_env or maintenance_prs.error) and maintenance_prs.maintenance_prs -%}
```

Reply:
This message is to be displayed if it's not a dimagi env AND if there are no errors, so it should be `and`

Reply:
The challenge is that the `not` is outside the brackets which invokes de Morgans law of boolean algebra:

https://ryanstutorials.net/boolean-algebra-tutorial/boolean-algebra-laws.php#demorgan

```
NOT(p AND k) = NOT(p) OR NOT(k)

and

NOT(p OR k) = NOT(p) AND NOT(k)
```

I think the expanded logic should be (correct me if this is wrong):
```
not is_dimagi_env and not maintenance_prs.error and maintenance_prs.maintenance_prs
```

which means that the contracted expression is:
```
not (is_dimagi_env or maintenance_prs.error) and maintenance_prs.maintenance_prs
```

Reply:
With the latest commit this is moot but probably still useful to know